ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/openSUSE.yml

```diff
@@ -62,4 +65,7 @@ Test_Tool_Packages:
   - pulseaudio
   - xorg-x11
   - xorg-x11-devel
-  - glibc-locale                  # Internationalization tests
+  - glibc-locale
+  - gnutls
+  - libnss3.so
+  - mozilla-nss                 # Internationalization tests
```

Comment:
Should this not all be aligned with the locale package install?

Reply:
1.  This definitely doesnt impact any existing openssl install, its additional tools to allow the tests to run, builds and tests have been run via VPC
2.  I haven't specifically tested CentOS6 ( I have 7 ), I will do some additional testing on this
3.  I'll get the dockerstatic files updated appropriately, can possibly look to roll them out via the docker package update process.

Reply:
The packages install and the playbooks run OK on CentOS6, and the tests execute OK, the ./ssl-tests-openssl-client.sh tests run OK , but there are a couple of issues in the other two test suites ( which I've sent over to zzambers ) these are 

```
ssl-tests-gnutls-client.sh

IGNORED: SunJSSE/DTLSv1.2: DTLSv1.0 + TLS_EMPTY_RENEGOTIATION_INFO_SCSV
Exception in thread "main" java.lang.RuntimeException: java.lang.IllegalArgumentException: Unknown protocol: CTYPE-OPENPGP
	at SSLSocketTester.testConfigurations(SSLSocketTester.java:283)
	at SSLSocketTester.testProvider(SSLSocketTester.java:234)
	at SSLSocketTester.testProviders(SSLSocketTester.java:190)
	at Main.main(Main.java:30)
Caused by: java.lang.IllegalArgumentException: Unknown protocol: CTYPE-OPENPGP
	at GnutlsClient.getJavaProtoName(GnutlsClient.java:88)
	at GnutlsClient.getSupportedCiphers(GnutlsClient.java:125)
	at SSLSocketTester.testConfigurations(SSLSocketTester.java:268)
	... 3 more
../ssl-tests/ssl-tests.mk:48: recipe for target 'ssl-tests-run' failed

```

And 

```
ssl-tests-nss-client.sh

../ssl-tests/ssl-tests.mk:48: recipe for target 'ssl-tests-run' failed
make: *** [ssl-tests-run] Error 1

FAILED: SunJSSE/TLSv1.3: TLSv1.3 + TLS_AES_256_GCM_SHA384
Sep 27, 2023 9:53:58 AM SSLSocketServer$1 run
SEVERE: null
javax.net.ssl.SSLHandshakeException: Client requested protocol TLSv1.2 is not enabled or supported in server context
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:312)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.negotiateProtocol(ClientHello.java:871)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.onClientHello(ClientHello.java:823)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.consume(ClientHello.java:801)
	at java.base/sun.security.ssl.SSLHandshake.consume(SSLHandshake.java:396)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:480)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:458)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:201)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1510)
	at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1425)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:455)
	at java.base/sun.security.ssl.SSLSocketImpl.ensureNegotiated(SSLSocketImpl.java:925)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:1016)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:976)
	at SSLSocketServer.serverLoop(SSLSocketServer.java:133)
	at SSLSocketServer$1.run(SSLSocketServer.java:75)
	at java.base/java.lang.Thread.run(Thread.java:833)

stderr: tstclnt: read from socket failed: SSL_ERROR_PROTOCOL_VERSION_ALERT: Peer reports incompatible or unsupported protocol version.
Sep 27, 2023 9:53:58 AM SSLSocketTester testConfiguration
SEVERE: null
java.lang.RuntimeException: Program exit value not zero: 1
	at ExternalClient.test(ExternalClient.java:267)
	at SSLSocketTester.testConfiguration(SSLSocketTester.java:392)
	at SSLSocketTester.testConfigurations(SSLSocketTester.java:322)
	at SSLSocketTester.testProvider(SSLSocketTester.java:234)
	at SSLSocketTester.testProviders(SSLSocketTester.java:190)
	at Main.main(Main.java:30)

FAILED: SunJSSE/TLSv1.3: TLSv1.3 + TLS_AES_128_GCM_SHA256
Sep 27, 2023 9:53:58 AM SSLSocketServer$1 run
SEVERE: null
javax.net.ssl.SSLHandshakeException: Client requested protocol TLSv1.2 is not enabled or supported in server context
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)
	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:365)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:321)
	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:312)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.negotiateProtocol(ClientHello.java:871)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.onClientHello(ClientHello.java:823)
	at java.base/sun.security.ssl.ClientHello$ClientHelloConsumer.consume(ClientHello.java:801)
	at java.base/sun.security.ssl.SSLHandshake.consume(SSLHandshake.java:396)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:480)
	at java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:458)
	at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:201)
	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:172)
	at java.base/sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1510)
	at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1425)
	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:455)
	at java.base/sun.security.ssl.SSLSocketImpl.ensureNegotiated(SSLSocketImpl.java:925)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:1016)
	at java.base/sun.security.ssl.SSLSocketImpl$AppInputStream.read(SSLSocketImpl.java:976)
	at SSLSocketServer.serverLoop(SSLSocketServer.java:133)
	at SSLSocketServer$1.run(SSLSocketServer.java:75)
	at java.base/java.lang.Thread.run(Thread.java:833)

stderr: tstclnt: read from socket failed: SSL_ERROR_PROTOCOL_VERSION_ALERT: Peer reports incompatible or unsupported protocol version.
Sep 27, 2023 9:53:58 AM SSLSocketTester testConfiguration
SEVERE: null
java.lang.RuntimeException: Program exit value not zero: 1
	at ExternalClient.test(ExternalClient.java:267)
	at SSLSocketTester.testConfiguration(SSLSocketTester.java:392)
	at SSLSocketTester.testConfigurations(SSLSocketTester.java:322)
	at SSLSocketTester.testProvider(SSLSocketTester.java:234)
	at SSLSocketTester.testProviders(SSLSocketTester.java:190)
	at Main.main(Main.java:30)

```

Reply:
Fixed the errant comment